src/compiler/checker.ts

```diff
@@ -16718,7 +16718,7 @@ namespace ts {
                     }
                 });
 
-                // ECMA-262 11.1.5 Object Initialiser
+                // ECMA-262 11.1.5 Object Initializer

```

Comment:
Happy to fix that, but it's not a spelling mistake. And amusingly, I've at least (accidentally) fixed the spelling to match the spec... (I wonder how it got out of sync...)